drivers/regulator/Kconfig.rpi_pico

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2023 TOKITA Hiroshi <tokita.hiroshi@fujitsu.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config REGULATOR_RPI_PICO
+	bool "Fake regulator driver"
+	default y
+	depends on DT_HAS_RASPBERRYPI_CORE_SUPPLY_REGULATOR_ENABLED
+	help
+	  Enable support for the FFF-based fake regulator driver.
+
+if REGULATOR_RPI_PICO
+
+config REGULATOR_RPI_PICO_INIT_PRIORITY
+	int "NXP RPI_PICO regulator driver init priority"
```

Comment:
Description and help string need to be updated, unless the RPI is using an NXP regulator (from the datasheet, this seems to be an internal regulator)

Reply:
Oops, this is embarrassing...
Fixed it.